.ci/jenkins/config/branch.yaml

```diff
@@ -1,78 +1,110 @@
-environment:
-  quarkus:
-    main:
-      enabled: true
-    branch:
-      enabled: true
-      version: '2.16'
-    lts:
-      enabled: true
-      version: '2.13'
+environments:
   native:
-    enabled: true
+    env_vars:
+      NATIVE: true
+      BUILD_MVN_OPTS_CURRENT: -Dnative -Dquarkus.native.container-build=true
+      ADDITIONAL_TIMEOUT: 720
+    ids:
+    - native
   mandrel:
-    enabled: true
-    builder_image: quay.io/quarkus/ubi-quarkus-mandrel:22.3-java17
-  mandrel_lts:
-    enabled: true
-    builder_image: quay.io/quarkus/ubi-quarkus-mandrel:22.3-java17
-    quarkus_version: 2.13
-  runtimes_bdd:
-    enabled: true
-productized_branch: true
+    env_vars:
+      NATIVE: true
+      BUILD_MVN_OPTS_CURRENT: -Dnative -Dquarkus.native.container-build=true
+      QUARKUS_NATIVE_BUILDER_IMAGE: mandrel
+      ADDITIONAL_TIMEOUT: 720
+    ids:
+    - native
+    - prod
+  quarkus-main:
+    env_vars:
+      QUARKUS_BRANCH: main
+    ids:
+    - quarkus
+  quarkus-branch:
+    env_vars:
+      QUARKUS_BRANCH: '2.16'
+    ids:
+    - quarkus
+  quarkus-lts:
+    env_vars:
+      BUILD_MVN_OPTS: -Dproductized -Ddata-index-ephemeral.image=quay.io/kiegroup/kogito-data-index-ephemeral
+      QUARKUS_BRANCH: '2.13'
+    ids:
+    - quarkus
+    - lts
+    - prod
+  mandrel-lts:
+    env_vars:
+      NATIVE: true
+      BUILD_MVN_OPTS: -Dproductized -Ddata-index-ephemeral.image=quay.io/kiegroup/kogito-data-index-ephemeral
+      BUILD_MVN_OPTS_CURRENT: -Dnative -Dquarkus.native.container-build=true
+      QUARKUS_BRANCH: '2.13'
+      QUARKUS_NATIVE_BUILDER_IMAGE: mandrel
+      ADDITIONAL_TIMEOUT: 720
+    ids:
+    - native
+    - prod
+    - lts
+  kogito-bdd:
+    auto_generation: false
+    ids:
+    - cloud
```

Comment:
Guessing this is not necessary for OptaPlanner?

Reply:
indeed will remove that one